src/assert.rs

```diff
@@ -97,6 +97,20 @@ impl Assert {
     }
 
     /// Clarify failures with additional context.
+    ///
+    /// # Examples
+    ///
+    /// ```rust
+    /// use assert_cmd::prelude::*;
+    ///
+    /// use std::process::Command;
+    ///
+    /// Command::main_binary()
+    ///     .unwrap()
+    ///     .assert()
+    ///     .append_context("main", "no args")
+    ///     .success();
+    /// ```
     pub fn append_context<D>(mut self, name: &'static str, context: D) -> Self
```

Comment:
Didn't have an  example on this because its mostly targeted at library authors (like say someone plugging in `duct`) but I can go either way.

Reply:
I only added an example because it may not be obvious that you can pass in two string slices here :)